internal/cmdexport/pathutil/pathutil_test.go

```diff
@@ -80,9 +80,9 @@ func TestIsInsideDirectory(t *testing.T) {
 	)
 
 	_, err := IsInsideDir("resources", ".")
-	assert.Error(t, err, "argument `path` (resources) is not an absolute path")
+	assert.Error(t, err, "argument `path` ('resources') is not an absolute path")
```

Comment:
Do we need quotes around resources here ?

Reply:
Removed.